.github/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM postgres:14
```

Comment:
do you want to copy this to Sophon as well so docker-compose spins up a rikai-pg enabled postgres?

Reply:
yep, we will build a docker image for services.